File: server/EntityLayer/Usuario/UsuarioEntity.py
from datetime import datetime

class UsuarioEntity:
    UsuarioId : int
    PersonaId: int
    CodUsuario: str
    Contrasena :str
    FechaVigencia: datetime
    FechaRegistro: datetime
    UsuarioRegistro: str
    EstadoRegistro: bool

    def Cargar(_DB: any):
        c = UsuarioEntity()
        c.UsuarioId = _DB['UsuarioId']
        c.PersonaId = _DB['PersonaId']
        c.CodUsuario = _DB['CodUsuario']
        c.Contrasena = _DB['Contrasena']
        c.FechaVigencia = _DB['FechaVigencia']
        c.FechaRegistro = _DB['FechaRegistro']
        c.UsuarioRegistro = _DB['UsuarioRegistro']
        c.EstadoRegistro = bool(_DB['EstadoRegistro'])
        return c

    def CargarAcceso(_DB: any):
        c = UsuarioEntity()
        c.PersonaId = _DB['PersonaId']
        # c.FEC_VIG = datetime.strptime(_DB['FEC_VIG'], "%Y-%m-%d %H:%M:%S")
        c.UsuarioRegistro = _DB['UsuarioRegistro']
        c.EstadoRegistro = bool(_DB['EstadoRegistro'])
        return c

File: Facturacion/FactCore/FactCore.EntityLayer/UsuarioEntity.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace FactCore.EntityLayer
{
    public class UsuarioEntity: BaseEntityObject
    {

        public UsuarioEntity()
        {
            this.UsuarioId = 0;
            this.CodUsuario = String.Empty;
            this.Contrasena = String.Empty;
            this.FechaVigencia = DateTime.Now;
            this.Estado = false;
        }

        public Int32 UsuarioId { get; set; }
        public String CodUsuario { get; set; }
        public String Contrasena { get; set; }
        public DateTime FechaVigencia { get; set; }
        public Boolean Estado { get; set; }
    }
}
